### PR TITLE
Add tests for hyphenated variable names

### PR DESCRIPTION
**Pull request by kaicataldo**

### CODE REVIEW

File: test/parser/index.js [hunk 1]
```diff
@@ -341,9 +341,10 @@ describe('parser', function() {
     describe('Focss variables', function() {
       const styles = `$foo: 600;
                       $bar: '#123ABC';
+                      $foo-bar: 'px';
```
Comment (kaicataldo) [inline test/parser/index.js]: Also added tests to ensure hyphenated  variable names are parsed correctly.

**Comment by weikinhuang** [comment]
:+1:

**Comment by weikinhuang** [comment]
But I think we should just stick to valid css comment styles

**Comment by kaicataldo** [comment]
Alright - I'm fine with sticking to standard CSS comments for now. Might as well add these tests in though

**Comment by hzoo** [comment]
👍

**Pull request merged**